api/nodes/serializers.py

```diff
@@ -72,8 +72,8 @@ class NodeSerializer(JSONAPISerializer):
     id = IDField(source='_id', read_only=True)
     type = TypeField()
 
-    category_choices = Node.CATEGORY_MAP.keys()
-    category_choices_string = ', '.join(["'{}'".format(choice) for choice in category_choices])
+    category_choices = Node.CATEGORY_MAP.items()
```

Comment:
Why? http://www.django-rest-framework.org/api-guide/fields/#choicefield
> choices - A list of valid values, or a list of (key, display_name) tuples.

Seems to me we weren't taking advantage of the fact that you could have both a value and a display name there.